translate/public/locale/en-US/translate.ftl

```diff
@@ -570,75 +570,35 @@ otherlocales-Translation--header-link =
     .title = Open string in { $locale } ({ $code })
 
 
-## Placeable parsers
+## Message terms
 ## Used to mark specific terms and characters in translations.
 
-placeable-parser-altAttribute =
-    .title = 'alt' attribute inside XML tag
-placeable-parser-camelCaseString =
-    .title = Camel case string
-placeable-parser-emailPattern =
+term-cli-option =
```

Comment:
Why are the strings prefixed with "term"? We already use the expression "term" for something else and we have a term module with [corresponding term-* entries](https://github.com/mozilla/pontoon/blob/2ac367a4bfafcac22b31687fa1b26d29926232f9/translate/public/locale/en-US/translate.ftl#L663-L669) in the FTL file.

I suggest we use highlight and follow the naming convention from the top of the file.